app/controllers/AdminUsers/rewards_controller.rb

```diff
@@ -1,7 +1,7 @@
 module AdminUsers
   class RewardsController < BaseController
     def index
-      render :index, locals: { rewards: Reward.order(title: :asc).includes([:category]) }
+      render :index, locals: { rewards: Reward.order(title: :asc).includes([:category, [photo_attachment: :blob]]) }
```

Comment:
```suggestion
      rewards: Reward.order(title: :asc).with_attached_photos.includes([:category]) }
```
There is method to help fix n+1 while attaching photos that looks better than this include :)
https://api.rubyonrails.org/classes/ActiveStorage/Attached/Model.html#method-i-has_many_attached

Reply:
Thanks a lot! 👍  I changed this :)